### ISSUE TITLE
METRICS_TOKEN should be optional

### ISSUE THREAD

**Issue by klausenbusk** [cdnjs/tools#69]
`autoupdate` can't be run locally without `-no-update` currently (and I don't want to disable the git commit/push logic) .
```
2020/07/04 20:38:41 f/fabric.js.json: run /usr/bin/git commit -m Add fabric.js v4.0.0-rc.1
panic: Env METRICS_TOKEN is missing
 [recovered]
	panic: Env METRICS_TOKEN is missing


goroutine 1 [running]:
github.com/cdnjs/tools/sentry.PanicHandler()
	/home/kristian/Hentet/cdnjs/tools/sentry/sentry.go:28 +0x74
panic(0x79c600, 0xc00016a8b0)
	/usr/lib/go/src/runtime/panic.go:969 +0x166
github.com/cdnjs/tools/util.GetEnv(0x82662c, 0xd, 0xc0004dfcf0, 0x5267e1)
	/home/kristian/Hentet/cdnjs/tools/util/env.go:18 +0xf9
github.com/cdnjs/tools/metrics.reportMetric(0x825cf1, 0xb)
	/home/kristian/Hentet/cdnjs/tools/metrics/metrics.go:17 +0x48
github.com/cdnjs/tools/metrics.ReportNewVersion(...)
	/home/kristian/Hentet/cdnjs/tools/metrics/metrics.go:25
main.commitNewVersions(0x8b5d60, 0xc00027e960, 0xc000141650, 0x1, 0x1)
	/home/kristian/Hentet/cdnjs/tools/cmd/autoupdate/main.go:225 +0x31a
main.main()
	/home/kristian/Hentet/cdnjs/tools/cmd/autoupdate/main.go:109 +0x65c
```

**Comment by xtuc** [comment]
Yes indeed. I suggest that we simply disable metrics if the token is not present (maybe just log something for debugging purposes).

**Comment by tc80** [comment]
Making changes now

**Comment by klausenbusk** [comment]
Fixed by #71
